package.json

```diff
@@ -38,6 +38,8 @@
 		"@babel/preset-react": "7.0.0",
 		"@babel/preset-typescript": "7.3.3",
 		"@cybozu/eslint-config": "4.0.1",
+		"@mdi/js": "3.6.95",
+		"@mdi/light-js": "^0.2.63",
```

Comment:
if you use "@mdi/light-js" it should be added to dependencies and not devDependencies since our package supports ESM usage.
Thanks!